EIPS/eip-3475.md

```diff
@@ -17,39 +16,37 @@ This API standard allows the creation of tokenized obligations with abstract on-
 
 ## Motivation
 
-
 Current LP tokens are simple [EIP-20](./eip-20.md) tokens with no complex data structure. To allow more complex reward and redemption logic to be stored on-chain, we need a new token standard that can manage:
 
  - Multidimensional token ID.
  - Store on-chain metadata.
  - Non-specified and flexible storage.
  - Gas efficient.
 
-This proposal allows the creation of any obligation with the same interface. It will enable any 3rd party wallet applications or exchanges to read these tokens' balance and redemption conditions. These bonds can also be batched as tradeable instruments. Those instruments can then be divided and exchanged in secondary markets.
+This proposal allows the creation of any obligation with the same interface. It will enable any 3rd party wallet applications or exchanges to read these tokens balance and redemption conditions. These bonds can also be batched as tradeable instruments. Those instruments can then be divided and exchanged in secondary markets.
```

Comment:
```suggestion
This EIP allows the creation of any obligation with the same interface. It will enable any 3rd party wallet applications or exchanges to read these tokens' balance and redemption conditions. These bonds can also be batched as tradeable instruments. Those instruments can then be divided and exchanged in secondary markets.
```
```suggestion
This proposal allows the creation of any obligation with the same interface. It will enable any 3rd party wallet applications or exchanges to read these tokens' balance and redemption conditions. These bonds can also be batched as tradeable instruments. Those instruments can then be divided and exchanged in secondary markets.
```
`This EIP` is a nit, but `tokens'` is required. Pick your poison.

Reply:
i included your suggestion